pkgs/applications/networking/nextcloud-client/default.nix

```
@@ -26,6 +26,9 @@
 
 stdenv.mkDerivation rec {
   pname = "nextcloud-client";
+  # make sure an official release has been published on
+  # https://github.com/nextcloud/desktop/releases and re-check the hash
```

```suggestion
  # https://github.com/nextcloud-releases/desktop/releases and re-check the hash
```
I think this is the more official release location that represents a green light on their self updater.